integration-tests/src/test/java/uk/gov/di/authentication/queuehandlers/SpotResponseIntegrationTest.java

```diff
@@ -82,7 +83,8 @@ void shouldDeleteIdentityCredentialFromDBForInvalidResponse() {
                         PERSISTENT_SESSION_ID,
                         REQUEST_ID,
                         CLIENT_ID);
-        identityStore.addAdditionalClaims(pairwiseIdentifier.getValue(), emptyMap());
+        identityStore.saveIdentityClaims(
+                pairwiseIdentifier.getValue(), emptyMap(), VOT.getValue(), null);
```

Comment:
should probably be `LevelOfConfidence.MEDIUM_LEVEL.getValue()` rather than vot.getvalue